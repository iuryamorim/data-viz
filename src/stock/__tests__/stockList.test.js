import React from 'react'
import TestRenderer from 'react-test-renderer'

import StockList from '../stockList'


describe('StockList', () => {
  it('render component', () =>  {
    const testRenderer = TestRenderer.create(<StockList />).toJSON()
    expect(testRenderer).toMatchSnapshot()
    expect(testRenderer.type).toEqual('div')
  })
})
