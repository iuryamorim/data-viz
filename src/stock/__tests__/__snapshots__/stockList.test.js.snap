// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StockList render component 1`] = `<div />`;
